src/dashboard/src/requirements/base.txt

```diff
@@ -8,12 +8,17 @@ django-extensions==1.1.1
 django-autoslug==1.7.1
 django-annoying==0.7.7
 elasticsearch>=1.0.0,<2.0.0
+gearman==2.0.2
 gunicorn==19.4.5
 lazy-paged-sequence
+git+git://github.com/artefactual-labs/agentarchives.git
```

Comment:
A question about this , I don't think this dependency is required for Archivematica 1.5.x.  It will be required for 1.6.0, but for this PR, merging into stable/1.5.x I think this line should be removed.

When doing the 1.6.0 release, I think we will want to pull a specific version of agentarchives from pypi, rather than link to a branch in github.